waspc/cli/src/Wasp/Cli/Command/Test.hs

```diff
@@ -0,0 +1,49 @@
+module Wasp.Cli.Command.Test
+  ( test,
+  )
+where
+
+import Control.Concurrent.Async (race)
+import Control.Concurrent.MVar (newMVar)
+import Control.Monad.Except (throwError)
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (Abs, Dir, (</>))
+import StrongPath.Types (Path')
+import Wasp.Cli.Command (Command, CommandError (..))
+import Wasp.Cli.Command.Common (findWaspProjectRootDirFromCwd)
+import Wasp.Cli.Command.Compile (compile)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import Wasp.Cli.Command.Watch (watch)
+import qualified Wasp.Cli.Common as Common
+import Wasp.Lib (ProjectRootDir)
+import qualified Wasp.Lib as Lib
+import qualified Wasp.Message as Msg
+
+test :: [String] -> Command ()
+test [] = throwError $ CommandError "Not enough arguments." "Expected: wasp test client <args>"
+test ("client" : args) = watchAndtest $ Lib.testWebApp args
+test ("server" : _args) = throwError $ CommandError "Invalid arguments." "Server testing not yet implemented."
```

Comment:
Are we good with "client" and "server"? I know we sometimes call it web app too, but I don't think we have a definitive term used and client felt more natural/shorter here.